.github/pull_request_template.md

```diff
@@ -16,7 +16,40 @@
 - [ ] Add a description of your pull request and instructions for the reviewer to verify your work.
 - [ ] If your pull request is for a specific ticket, link to it in the description.
 - [ ] Stay on point and keep it small so the merge request can be easily reviewed.
-- [ ] Consider adding unit tests, especially for bug fixes. If you don't, tell us why.
 - [ ] Tests and linting passes.
-- [ ] Consider updating documentation. If you don't, tell us why.
-- [ ] If relevant, list the environments / browsers in which you tested your changes.
+
+#### Unit tests
+
+- [ ] Added
+- [ ] Not required
+
+#### Documentation
+
+- [ ] Updated
+- [ ] Not required
+
+#### Browser testing
+
+- [ ] I have tested in the following browsers and environments:
+- [ ] Not required
+
+#### Data protection
+
+- [ ] Not relevant
+- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly
+
+#### Accessibility
+
+- [ ] Automated WCAG 2.1 tests pass
+- [ ] Manual WCAG 2.1 tests completed
+- [ ] I have tested in a screen reader
+- [ ] I have tested in high-contrast mode
+- [ ] Not required
+
+#### Sustainability
+
+- [ ] Images are optimised and lazy-loading used where appropriate
+- [ ] Perfomance and transfer of data considered
```

Comment:
Question: As a FE developer what would I be testing here? Sizes of assets?

Reply:
I think so - hoping @Morsey187 might have some more concrete suggestions. I'm going to apply your other suggestions and merge so we can start using this, but can amend again in future if there are any further changes Ben suggests.